featurehub-javascript-client-sdk/CHANGELOG.md

```diff
@@ -1,3 +1,8 @@
+#### 1.1.6
+- Change to interfaces instead of classes for models, and match the OpenAPI current definition
+- Add support for deregistering feature and readiness listeners (ready for React SDK)
+- Deprecate the readyness listener for a readiness listener
+- Add support for stale environments
```

Comment:
we need to link to GitHub PR for these changes. I presume it's in this PR?

Reply:
er, i don't understand this comment

Reply:
we provide the link to the PR with the change. See in the previous version release notes for an example.